Dockerfile.alpine

```diff
@@ -9,8 +9,6 @@ RUN \
   mkdir -p /opt/distillery-test && \
   mix local.rebar --force && \
   mix local.hex --force && \
-  pushd /opt && \
-  git clone https://github.com/bitwalker/distillery-test && \
-  popd
+  git clone https://github.com/bitwalker/distillery-test
 
-CMD ["bin/bash"]
```

Comment:
Can you change this back, `bash` is required for Distillery, and I prefer using it by default. This image is for my own local testing anyway. Other than that 👍

Reply:
Thank you for your review.
I fixed it to full path to `bash`.